sunpy/data/_sample.py

```diff
@@ -51,52 +65,58 @@
     "NORH_LIGHTCURVE": ("tca110810", "")
 }
 
-sample_files = {}
-for key in _files:
-    sample_files[key] = os.path.abspath(os.path.join(sampledata_dir, _files[key][0]))
+
+free_required=145227776
```

Comment:
Something is already unsatisfying about this approach if there needs to be a magic number.